core/templates/self_list.h

```diff
@@ -77,7 +77,6 @@ class SelfList {
 		}
 
 		void remove(SelfList<T> *p_elem) {
-			ERR_FAIL_COND(p_elem->_root != this);
```

Comment:
This should *not* be removed, it is required to prevent incorrectly removing nodes from the wrong list

Reply:
It *could* be replaced with just:
```cpp
if (p_elem->root != this) {
	return;
}
```
But I don't think this has any part of this crash, unless again there's some invalid code trying to remove a node from the wrong list